recipe/base.rb

```diff
@@ -44,4 +44,16 @@ def archive_path_name
   def tmp_path
     "/tmp/#{@host}/ports/#{@name}/#{@version}"
   end
+
+  def download_file(url, full_path, count = 3)
```

Comment:
Is this used somewhere? I can't find anything related in these changes.

Reply:
[Got it](https://github.com/flavorjones/mini_portile/blob/master/lib/mini_portile2/mini_portile.rb#L429)